birdhouse/components/monitoring/config/magpie/config.yml.template

```diff
@@ -1,20 +1,26 @@
 providers:
   grafana:
-    url: http://grafana:3000
+    # below URL is only used to fill in the required location in Magpie
+    # actual auth validation is performed with Twitcher 'verify' endpoint without accessing this proxied URL
+    url: http://proxy:80
     title: Grafana
     public: true
     c4i: false
     type: api
     sync_type: api
   prometheus:
-    url: http://prometheus:9090
+    # below URL is only used to fill in the required location in Magpie
+    # actual auth validation is performed with Twitcher 'verify' endpoint without accessing this proxied URL
+    url: http://proxy:80
     title: Prometheus
     public: true
     c4i: false
     type: api
     sync_type: api
   alertmanager:
-    url: http://alertmanager:9093
+    # below URL is only used to fill in the required location in Magpie
+    # actual auth validation is performed with Twitcher 'verify' endpoint without accessing this proxied URL
+    url: http://proxy:80
```

Comment:
Although the `/twitcher/ows/verify` endpoint approach is used, there should not be any need to remove the actual endpoint where it could be reached if we desired to use the `/twitcher/ows//proxy` endpoint, or opened the port, for debugging or similar.

This is what is currently done for other services. 
For example, `flyingpigeon` uses `http://${PAVICS_FQDN}:8093/wps` although `http` and `8093` are not directly exposed, but it can be done by docker compose overrides.

Reply:
@fmigneault just to clarify, you're suggesting we change the `url: http://proxy:80` back to what it was before so that it's easier to change back to the proxy path if needed?

I'm happy to do that, but I'm a little concerned that this will encourage people to use the "proxy" method instead of the "verify" method which (as we saw previously) makes these components inaccessible from the UI.

Reply:
Yes. Thanks.
I'm not worried about them using this approach. We can always recommend the verify method if a PR is submitted. It is a relatively easy switch.